lib/plugins/physics.js

```diff
@@ -117,17 +138,25 @@ function inject (bot, { physicsEnabled }) {
     return dYaw
   }
 
-  function updatePosition (dt) {
-    // If you're dead, you're probably on the ground though ...
-    if (!bot.isAlive) bot.entity.onGround = true
+  // returns false if packet should be sent, true if not
+  function sendPositionPacketInDeath () {
```

Comment:
This seems like quite ambigious naming

Reply:
I think its a good name. Better then it was before. And it does not return where there the bot is dead but if the bot is dead and has been dead for a while. Do you have a better suggestion?

Reply:
The function does the opposite of it's name.

My best suggestion is either call it shouldNotSendPosition or invert its return values